spec/support/matchers.rb

```diff
@@ -323,9 +323,9 @@
       actual['name'] == expected.name &&
       actual['state'] == expected.state.to_s &&
       actual['managed_by_atmosphere'] == expected.managed_by_atmosphere &&
-      actual['compute_site_id'] == expected.tenant_id &&
       actual['appliance_type_id'] == expected.appliance_type_id &&
-      actual['architecture'] == expected.architecture.to_s
+      actual['architecture'] == expected.architecture.to_s &&
+        (expected.tenants.collect{|t| t.id}.include? actual['compute_site_id'])
```

Comment:
Prefer `map` over `collect`.<br>Use 2 (not 4) spaces for indenting an expression spanning multiple lines.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Pass `&:id` as an argument to `collect` instead of a block.